src/main/resources/assets/nguhcraft/lang/de_de.json

```diff
@@ -0,0 +1,282 @@
+  "item.nguhcraft.thou_hast_been_yeeten_crossbow_version": "Du wurdest geyeeted (Armbrustversion)",
+  "item.nguhcraft.trident_of_the_seven_winds": "Dreizack der Sieben Winde",
+  "item.nguhcraft.veneficium_armour_trim_smithing_template": "Venēficium-Rüstungsbesatz",
+  "item.nguhcraft.wrath_of_zeus": "Zorn des Zeus",
+
+  "itemGroup.nguhcraft.treasures": "Schätze",
+
+  "jukebox_song.nguhcraft.nguhrovision_2024": "Marcy Schiekah – Spiegelung",
+
+  "lore.nguhcraft.ancient_drop_of_cherry": "\u0001Ein Schluck der Schnitzel der Wände\nder Plattenbörse.\n\nAls Rabo die Antiken unterirdisch verbannte,\nweinte er, und seine Tränen spränkelten\nden Kirschbaum.\nObwohl das heutige Kirschholz nicht länger\nseine Kraft trägt, verbleibt jedoch\nein Hauch davon in den Amaranthwänden\nder Börse.",
+
+  "multiplayer.disconnect.muted": "Du bist derzeit auf dem Discordserver von Agma Schwa stummgeschalten",
+
+  "nguhcraft.block.locked": "%1$s ist verschlossen (Schlüssel: %2$s)!",
+  "nguhcraft.door": "Tür",
+
+  "painting.nguhcraft.chillvana_metro.author": "sq",
+  "painting.nguhcraft.chillvana_metro.title": "Chillvana Metro-Diagramm",
+  "painting.nguhcraft.gambianholiday.author": "Agma Schwa",
+  "painting.nguhcraft.gambianholiday.title": "???",
+  "painting.nguhcraft.gold_nguh.author": "Ætérnal",
+  "painting.nguhcraft.gold_nguh.title": "Goldenes Nge",
+  "painting.nguhcraft.gold_nguh_small.author": "Chronos",
+  "painting.nguhcraft.gold_nguh_small.title": "Goldenes Nge",
+  "painting.nguhcraft.great_wave.author": "Hokusai",
+  "painting.nguhcraft.great_wave.title": "Die Große Welle",
+  "painting.nguhcraft.kozdenen_rail_diagram.author": "total pleb",
+  "painting.nguhcraft.kozdenen_rail_diagram.title": "Kozdenen Schienendiagramm",
+  "painting.nguhcraft.leshrail_diagram.author": "theidealist",
+  "painting.nguhcraft.leshrail_diagram.title": "Leschschienendiagramm",
+  "painting.nguhcraft.map.author": "Marcy Schiekah",
+  "painting.nguhcraft.map.title": "Weltkarte",
+  "painting.nguhcraft.rabo.author": "Agma Schwa",
+  "painting.nguhcraft.rabo.title": "Rabo",
+  "painting.nguhcraft.rail_diagram.author": "sq",
+  "painting.nguhcraft.rail_diagram.title": "Schienendiagramm",
+  "painting.nguhcraft.rails_of_eras.author": "N/A",
+  "painting.nguhcraft.rails_of_eras.title": "Erasschienendiagramm",
+  "painting.nguhcraft.rauratoshan_loop.author": "Galaxy System: Hikari",
+  "painting.nguhcraft.rauratoshan_loop.title": "Rauratoshan Schleifendiagramm",
+
+  "subtitles.nguhcraft.nguh": "Nge",
+
+  "tag.item.nguhcraft.repairs_amethyst_armour": "Repariert Amethystrüstung und -werkzeuge",
+  "tag.item.nguhcraft.tinted_oak_logs": "Getönte Eichenholzstämme",
+
+  "trim_pattern.nguhcraft.atlantic": "Atlantisch-Rüstungsbesatz",
+  "trim_pattern.nguhcraft.cenrail": "Zenschienen-Rüstungsbesatz",
+  "trim_pattern.nguhcraft.ice_cold": "Eiskalt-Rüstungsbesatz",
+  "trim_pattern.nguhcraft.veneficium": "Venēficium-Rüstungsbesatz",
+  "block.nguhcraft.example": "Example"
```

Comment:
Er, I don’t think that’s supposed to be here lol; that’s probably left over from whatever template you used